docs/reference/user-management.md

```diff
@@ -0,0 +1,107 @@
+# User management
```

Comment:
FYI there is new info at the bottom of this doc that could go under 'best practices': https://www.notion.so/n8n/P0-UM-Docs-notes-ec67f85a5f0d46c08c58ff0862926ba3

Plus it seems like not everything mentioned in the alpha testing doc is in here?

Reply:
The things missing from the alpha testing doc are the features you can turn off if you don't want them (tags, templates, survey). However, these are not specific to user management. I've added another tip linking to the environment variables list and calling out those options.